src/modules/dashboard/load.php

```diff
@@ -75,6 +68,26 @@ function () {
 		},
 		9
 	);
+
+	add_action(
+		'_wl_dashboard__main',
+		function () {
+			wp_enqueue_script( 'wl-angular-app' );
+			$iframe_src = WL_ANGULAR_APP_URL . '#dashboard-content';
+			// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+			echo "
+			<style>
+			    #wlx-plugin-app {
+			      margin-left: 20px;
+			      width: calc(100% + 20px);
+			      min-height: 1500px;
+			    }
+		    </style>
```

Comment:
@naveen17797 should this go in app? maybe not, just asking - if dashboard is the default layout